### ISSUE TITLE
Description: Consider "photons off"

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#291]
Currently, we have the following values for the Photo Rate slider:
- light source off
- very slow
- slow
- fast

So on first focus, screen reader users hear:
- Infrared Photon Rate, slider, light source off
 
Then when they move the slider, will hear the new values as they pass them:
- very slow
- slow
- fast

What do people think of changing "light source off" to:
- photons off

And if we change the off value to refer to photons (red underline in screen shot), do we need to make any other changes (see blue marks in screen shot) to "light sources" wording in the PDOM?

![light-sources](https://user-images.githubusercontent.com/1902116/75473650-24fd2300-5970-11ea-9f18-58e6c01a62ce.png)

**Comment by terracoda** [comment]
And if the string is changed, the slider would sound like this:
- Infrared Photon Rate, slider, photons off
 
Then as they the slider to an on-value:
- very slow
- slow
- fast

I think, "photons very slow" is more easily inferred after "photons off" than with "light source off".

**Comment by arouinfar** [comment]
@terracoda great idea! I think the "photons off" phrasing flows better with the rest of the descriptions.

**Comment by BLFiedler** [comment]
Some small, deep part of me thinks "off" should refer to the thing the photon is doing (emitting, emission, etc). However, I don't actually feel that strongly about it and agree that the consistency provided is nice!

**Comment by terracoda** [comment]
@BLFiedler, "photon emissions off" could work, too.

**Comment by BLFiedler** [comment]
I think the conciseness > absolute precision here. Stick to "photons off".

**Comment by terracoda** [comment]
@Matthew-Moore240, what do you think?

**Comment by Matthew-Moore240** [comment]
It looks good to me! I think it gives a clear idea of whats going on.

**Comment by terracoda** [comment]
@jessegreenberg, please change the off value for the photon rate slider from:
- light source off

TO:
- photons off

**Comment by terracoda** [comment]
The end result should be something like:
- Infrared Photon Rate, slider, photons off

Then when moving the slider:
- very slow
- slow
- fast

**Comment by terracoda** [comment]
Due to issues in interviews the slider is being changed to a button in #295.
This issue is no longer relevant. Work on this object will continue in #295 and other issues.
